packages/js/product-editor/src/components/section/style.scss

```diff
@@ -0,0 +1,16 @@
+.wp-block-woocommerce-product-section {
+    &__title {
+        margin-top: 64px;
```

Comment:
I believe on the figma the top-margin here is 48px between the section and the previous element (separator). And should this be attached to the section container instead of the title?

Reply:
Looks like the one linked in the original issue is dated.  I found what I think is the latest manually under `MVP 0.5 March '23`.  Is that the one you're seeing?

Reply:
Hey @joshuatf , I have just been referencing the one linked to in the issue. I'm a little confused with Figma still, and don't see anything with `MVP 0.5 March '23` in that Figma file ; is it another one entirely? Still not sure how to navigate the designs to find the most current ones.

If on second glance the more current design aligns with what you have, then we can definitely just stick with those.